packages/configuration-accessibility/package.json

```diff
@@ -6,9 +6,6 @@
     "@hint/hint-axe": "^4.1.6"
   },
   "description": "webhint's axe-based accessibility configuration for WCAG 2.1 Level A and Level AA requirements",
-  "engines": {
```

Comment:
Doesn't we need this anymore?

Reply:
I'm leaving it only on entry packages (`hint`, `create-*`) so next time we drop support for a node version we don't have to touch everything.

Reply:
Although I should probably update the `create-*` templates